src/QMCWaveFunctions/LCAO/LCAOrbitalSet.h

```diff
@@ -50,6 +50,10 @@ struct LCAOrbitalSet : public SPOSet
 
   LCAOrbitalSet(const LCAOrbitalSet& in);
 
+#if defined(ENABLE_OFFLOAD)
+  bool isOMPoffload() const override { return true; }
+#endif
+
```

Comment:
I'm not sure if this alone (`#if defined(ENABLE_OFFLOAD)`) is sufficient for identifying when to do the `LCAOrbitalSet::mw_evaluate*` on the device for LCAOs.  I suspect I might need to add more logic there, but I was planning to take a closer look at that after the other bugs are fixed.

I think the `isOMPoffload()` here only affects whether the determinant passes host or device ptrs to the inverse psi rows, but I use the same logic in `LCAOrbitalSet::mw_evaluate*` for choosing whether to do the AO->MO gemm on host or device.